interfaces/AW-App/src/app/validation-components/validate-program/validate-program.component.ts

```diff
@@ -84,3 +254,29 @@ export class ValidateProgramComponent implements ValidationComponent {
   }
 
 }
+
+
+class Question {
```

Comment:
I'm not sure how much would make sens, but all these 'models' could be moved to separate files that can be (maybe) easier to keep in sync between the PA-app and the AW-app...

Reply:
Makes sense. But in the 'cutoff' between 'process feedback right away' and 'make new task', I think this one falls in the latter? And if agree, new task within this sprint/PBI? Or on backlog?